src/components/AddComment.jsx

```diff
@@ -0,0 +1,66 @@
+import { useState, useContext } from 'react';
+import { useParams } from "react-router-dom";
+import { postComment } from '../api';
+import { UserContext } from "../context/User";
+
+function AddComment({onAddComment}) {
+    const { article_id } = useParams();
+    const { user, setUser } = useContext(UserContext);
+    const [error, setError] = useState(null);
+    const [input, setInput] = useState({
+        body: "",
+        author: user[0].username
+    });
+
+   function handleChange(event){
+    const {name,value} = event.target
+      setInput({...input, [name]: value})
```

Comment:
I would probably just hard code this as body as currently it makes someone reading the code have to look around and find what name is